brjs-sdk/sdk/libs/javascript/br-i18n/test-unit/tests/TranslatorTest.js

```diff
@@ -411,4 +411,18 @@
 		assertEquals(translator.parseNumber('1000.50'), 1000.50);
 		assertEquals(translator.parseNumber('1,000,000'), 1000000);
 	};
+
+	TranslatorTest.prototype.test_registerTranslations = function() {
+		var translations = {
+			key: 'value',
+			upperCaseKey: 'Value'
+		};
+		var Translator = require('br/i18n/Translator');
+		var translator = new Translator(this.i18nTimeDateNumberMessages, 'locale');
+
+		translator.registerTranslations('locale', translations);
+
+		assertEquals(translator.getMessage('key'), 'value');
+		assertEquals(translator.getMessage('upperCaseKey'), 'Value');
+	};
```

Comment:
do we need a test that ensures this method doesn't override existing translations?